mingw-w64-git/git-wrapper.c

```diff
@@ -64,8 +64,19 @@ static void find_exe_realpath(LPWSTR exepath, int count) {
 	if (dwlen >= MAX_PATH) {
 		wcscpy(exepath, mdexe);
 	} else {
-		size_t offset = wcsncmp(L"\\\\?\\", realexe, 4) ? 0 : 4;
-		size_t length = dwlen - offset;
+		size_t offset = 0;
+		size_t length = dwlen;
+
+		if (!wcsncmp(realexe, L"\\\\?\\", 4)) {
```

Comment:
As someone who is looking at this with minimal context, a higher level comment about what this is trying to achieve might be helpful I can see we are looking to strip off the leading `\\?\`, but I am still reading to understand the why... will keep reading

Reply:
Good point, will add a comment.